components/brave_wallet_ui/components/shared/create-placeholder-icon/index.tsx

```diff
@@ -31,7 +31,7 @@ function withPlaceholderIcon (WrappedComponent: React.ComponentType<any>, config
 
     const bg = React.useMemo(() => {
       if (selectedAsset?.symbol !== 'ETH' && stripERC20TokenImageURL(selectedAsset?.logo) === '') {
-        return background({ seed: selectedAsset?.contractAddress.toLowerCase() })
+        return background({ seed: selectedAsset.contractAddress ? selectedAsset?.contractAddress.toLowerCase() : selectedAsset.name })
```

Comment:
I think we can do something like this:

```
selectedAsset?.contractAddress?.toLowerCase() ?? selectedAsset.name
```

Reply:
Unfortunately I tried this way and it will still return an empty string.. :/